python/python/lance/arrow.py

```diff
@@ -292,15 +311,18 @@ def tensorflow_metadata_decoder(images):
         )
 
     def to_tensor(
-        self, decoder: Optional[Callable[[pa.BinaryArray], np.ndarray]] = None
+        self,
+        decoder: Optional[
+            Callable[[pa.BinaryArray, pa.LargeBinaryArray], np.ndarray]
```

Comment:
Should this be `Callable[Union[pa.BinaryArray, pa.LargeBinaryArray]], np.ndarray]`?